module/data/activity/attack-data.mjs

```diff
@@ -98,11 +98,15 @@ export default class AttackActivityData extends BaseActivityData {
    * @type {Set<string>}
    */
   get availableAbilities() {
-    // Defer to item if available
-    if ( this.item.system.availableAbilities ) return this.item.system.availableAbilities;
+    const attackClassification = this.attack.type.classification;
+
```

Comment:
Not really running into any line-length issues, so I don't think it is necessary to give this its own local variable.

```suggestion
```